xrootd/client/main_test.go

```diff
@@ -0,0 +1,11 @@
+// Copyright 2018 The go-hep Authors.  All rights reserved.
+// Use of this source code is governed by a BSD-style
+// license that can be found in the LICENSE file.
+
+// +build xrootd_test_with_server
+
+package client // import "go-hep.org/x/hep/xrootd/client"
```

Comment:
is this file neeeded?
it doesn't seem to be needed at present.

Reply:
It defines an `Addr` flag, which is used by tests to connect to the remote XRootD server.  (locally I am running tests with `-tags xrootd_test_with_server --addr=ccxrootdgotest.in2p3.fr:9001`). That definition is placed in `main_test.go` because later there will be `protocol_test.go`, etc, and I'm not sure it will not be confusing to have that flag somewhere in one of these files. Anyway, if you think that it's ok to put it in `client_test.go`, I'll do that. :)

Reply:
ok. let's remove this file from this PR (and only add it as is in the PR that makes use of that `Addr` flag.)
I think I'll probably suggest another way to perform that test :)

Reply:
> (and only add it as is in the PR that makes use of that Addr flag.)

It's used in `client_test.go` already (see line 19): `client, err := NewClient(ctx, *Addr)`

Reply:
Ah. Sorry. That passed me.